reference/docs-conceptual/learn/ps101/00-introduction.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Introduction | PowerShell 101
+ms.date: 06/02/2020
+ms.topic: guide
+ms.custom: Contributor-mikefrobbins
+ms.reviewer: mirobb
+---
+# Introduction
+
+<table>
+  <tr><td>
+  <a href="https://leanpub.com/powershell101">
+  <img src="media/powershell101-150x194.png" alt="PowerShell 101 (the book)" />
+  </a>
+  </td>
+  <td colspan=2>
+  This content originally appeared in the book <em>PowerShell 101</em> by Mike F Robbins. We thank
+  Mike for granting us permission to reuse his content here. The content has been edited from the
+  original publication. You can still get the original book from Leanpub at
+  <a href="https://leanpub.com/powershell101">PowerShell 101</a>.
+  </td></tr>
+</table>
```

Comment:
Is there a reason this is in a HTML table instead of using the ::: column formatting?

Reply:
Yes. Image wasn't working properly.